FND/Places/Addresses.rdf

```diff
@@ -99,29 +120,457 @@
+						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+					</owl:Restriction>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:subClassOf>
+		<rdfs:label>conventional street address</rdfs:label>
+		<skos:definition>physical address that identifies a specific building and unit or location within a building on a street to which communications may be delivered</skos:definition>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-adr;Floor">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitDesignator"/>
+		<rdfs:label>floor</rdfs:label>
+		<skos:definition>all of the rooms or areas on the same level of a building; a story</skos:definition>
+		<fibo-fnd-plc-adr:requiresSecondaryUnitRange rdf:datatype="&xsd;boolean">true</fibo-fnd-plc-adr:requiresSecondaryUnitRange>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;GeographicDirectionalSymbol">
+		<rdfs:subClassOf rdf:resource="&lcc-lr;CodeElement"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;hasTag"/>
+				<owl:someValuesFrom rdf:resource="&xsd;string"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>geographic directional symbol</rdfs:label>
+		<skos:definition>code element that gives directional information for postal delivery</skos:definition>
+		<skos:example>In the United States, these include N, S, E, W, NE, NW, SE, SW.</skos:example>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;InternationalAddress">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-plc-cty;hasCountry"/>
+				<owl:onClass rdf:resource="&lcc-cr;Country"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>international address</rdfs:label>
+		<skos:definition>physical address located outside the country of origin</skos:definition>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-fnd-plc-adr;PhysicalAddress">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;Address"/>
+		<rdfs:subClassOf>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasPostalCode"/>
+						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
+						<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+					</owl:Restriction>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasPostcode"/>
+						<owl:onClass rdf:resource="&fibo-fnd-plc-adr;Postcode"/>
+						<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+					</owl:Restriction>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
```

Comment:
BusinessCenter is a subclass of Municipality, so it is allowed.  Anything from geonames could be used as a subclass of municipality, or an equivalence could be made so that this works.